src/reader/widgets/ReaderWidget.vue

```diff
@@ -57,6 +57,19 @@
           this.$parent.$el.scrollTop = 0;
         });
       },
+      versionMetadata: {
+        immediate: true,
+        handler() {
+          if (!this.versionMetadata) {
+            this.setVersionMetadata();
+          }
+          // @@@ determine if we want this hooked
+          // further into scaife-skeleton
+          document.querySelector(
```

Comment:
@paltman thoughts on adding a Portal for setting the header?

https://github.com/scaife-viewer/scaife-skeleton/blob/5d9034b349ed33ebe293756e721369fefa9497c7/src/skeleton/main/MainLayout.vue#L5

Or we could have a reactive property in Vuex e.g. `mainWidgetTitle`.

Reply:
@paltman suggest moving this down into `ReaderWidget`

Reply:
resolved via c8c47c6c0b46fe04bcf040c2fbaefc7e750418a3